src/login/components/UserProfileFormFields.svelte

```
@@ -0,0 +1,207 @@
+    for (const f of $formFieldStates) {
+        if (!desiredOrder.includes(f.attribute.name)) rest.push(f);
+    }
+
+    return [...ordered, ...rest];
+});
+
+
+</script>
+<div class="grid grid-cols-1 md:grid-cols-1 gap-x-4">
+  <div class="flex flex-col md:flex-row md:space-x-4">
+  {#each $sortedFormFieldStates.filter(f => f.attribute.name === 'firstName' || f.attribute.name === 'lastName') as formFieldState, i (i)}
```

ESLint

[error] Replace `··{`#each`·$sortedFormFieldStates.filter(f` with `····{`#each`·$sortedFormFieldStates.filter((f)`

(prettier/prettier)